classes/files_handler.rb

```diff
@@ -0,0 +1,34 @@
+require 'json'
+
+class Storage
+  @files_path = 'data/'
+  class << self
+    def path(filename)
+      "#{@files_path}#{filename}"
+    end
+
+    def open_or_create_file(filename)
+      File.open(path(filename), 'w')
+    end
+
+    def read_file_content(name)
+      filename = path(name)
+      return [] unless File.exist?(filename)
+
+      file = File.open(filename, 'r')
+      file_data = file.read
+      return [] if file_data.empty?
+
+      file_content = JSON.parse(file_data)
+      file.close
+      file_content
+    end
+
+    def save_file_content(filename, content)
+      file = open_or_create_file(filename)
+      json_string = JSON.dump(content)
+      file.write(json_string)
+      file.close
+    end
+  end
+end
```

Comment:
- [ ] I noticed that data are stored in the JSON files when I am still inputting data from the console. Based on the requirements, data should be saved on exit from the application and not during the process. Kindly adjust your logic to ensure that data is saved when a user exits the application. 👍 💻

Reply:
Hello @Gambit142 👋️, thank you for the suggestion but from the requirement, even though we were to persist the data in .json files, it wasn't specified that we should do that when a user exits the application🤗️.

![Screenshot from 2023-06-01 20-01-00](https://github.com/Stephen-Adom/catalog-of-things/assets/42853016/9b598cdb-2203-41fc-b435-5f291c5140d0)

Reply:
@Stephen-Adom, it's fine if it wasn't explicitly stated in your project requirement. It was stated in my guideline. You can skip this requirement. 👍 😎

Reply:
Hello @Gambit142👋️, thank you. We really appreciate the feedback 🙏️🤗️

Reply:
Hi @Stephen-Adom, you are welcome. Great work done 👏